src/coreclr/jit/ee_il_dll.cpp

```diff
@@ -1122,9 +1122,39 @@ void Compiler::eeDispLineInfos()
  * (e.g., host AMD64, target ARM64), then VM will get confused anyway.
  */
 
+void Compiler::eeAllocMem(AllocMemArgs* args, UNATIVE_OFFSET hotSizeRequest, UNATIVE_OFFSET coldSizeRequest)
+{
+#ifdef DEBUG
+    // Fake splitting implementation: hot section = hot code + 4K buffer + cold code
+    const UNATIVE_OFFSET buffer = 4096;
```

Comment:
nit: `buffer` is a pretty generic name. How about:
```suggestion
    const UNATIVE_OFFSET fakeSplittingBuffer = 4096;
```